spec/rails_helper.rb

```diff
@@ -11,6 +11,8 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
```

Comment:
- [ ] Good job by not using [the bullet gem](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Do%20not%20use,after%20the%20fix) while fixing the `n+1` problem. However , as required for the project, kindly ensure that you add screenshots of the logs `before` and `after` the fix to the PR description

Reply:
@tresorsawasawa  thanks for noticing, we are adding the images for `n+1`